web/src/components/Contact.tsx

```diff
@@ -17,7 +17,15 @@ const Contact = ({
   return (
     <Collapsible title={headline ?? t('contactInformation')}>
       <div>
-        {!!website && <ContactItem iconSrc={WebsiteIcon} iconAlt={t('website')} link={website} content={website} />}
+        {!!website && (
+          <ContactItem
+            iconSrc={WebsiteIcon}
+            iconAlt={t('website')}
+            link={website}
+            content={'Website'}
```

Comment:
🔧 Instead of using hardcoded labels, we use i18n to make the labels translatable in different languages.
Therefore you can search for `translations.json` in `translations` folder.
In the file you can add a key value pair in the namespace "pois" -> "en"
f.e. `"website" : "Website"` for english translation

To use this translation in the `Contact` component you can use this hook
```
const { t } = useTranslation('pois')
...
and pass `t('website')` as content value
```